cfdi.go

```diff
@@ -187,13 +189,9 @@ func validateSupport(inv *bill.Invoice) error {
 		}
 	}
 
-	if inv.Currency != currency.MXN {
-		errs["currency"] = ErrNotSupported
-	}
-
 	for _, l := range inv.Lines {
-		if l.Item.Currency != currency.CodeEmpty && l.Item.Currency != currency.MXN {
-			errs["line currency"] = ErrNotSupported
+		if l.Item.Currency != currency.CodeEmpty && l.Item.Currency != inv.Currency {
```

Comment:
I don't think this is necessary as GOBL will always ensure that the item's currency matches that of the document.

Reply:
Addressed on be25203. Thanks!